package.json

```diff
@@ -25,9 +25,9 @@
     "@babel/runtime": "^7.4.4",
     "@changesets/cli": "^2.0.3",
     "@changesets/get-github-info": "^0.2.1",
-    "@emotion/cache": "^10.0.9",
-    "@emotion/core": "^10.0.9",
-    "@emotion/css": "^10.0.9",
+    "@emotion/cache": "^11.0.0",
+    "@emotion/react": "^11.0.0",
```

Comment:
Would force a minimum for "@emotion/react" to "^11.1.1" (others are still 11.0.0). Emotion 11 is new, I wouldn't care.

If done, must be done in all package.json